client/my-sites/customer-home/controller.jsx

```diff
@@ -20,9 +20,20 @@ export default async function ( context, next ) {
 export function maybeRedirect( context, next ) {
 	const state = context.store.getState();
 	const slug = getSelectedSiteSlug( state );
+
 	if ( ! canCurrentUserUseCustomerHome( state ) ) {
 		page.redirect( `/stats/day/${ slug }` );
 		return;
 	}
+
+	const launchpadEnabled = true;
+	const launchpadFlow = 'newsletter';
```

Comment:
I imagine we should check our environment variable for launchpad here as well?

Reply:
Yeah totally agree 👍 Will do so in a bit